sig/transaction/constitution.md

```diff
@@ -43,16 +43,16 @@ The main responsibility of Transaction SIG is to discuss and plan the future dev
 ## SIG Active Rules, Promotion Mechanism
 
 - Assessment & Promotion System
-  - Tech Leader evaluates group members on a monthly basis to determine whether members can be promoted to Reviewer from Active Contributor:
+  - Tech Lead evaluates group members on a monthly basis to determine whether members can be promoted to Reviewer from Active Contributor:
     - Familiar with the code base
-    - Nominated by at least 2 TiKV Committers
+    - Nominated by at least 2 Committers
     - PR contributions meet any of the following:
       - Merged transaction-related PRs totaled more than 10
       - Propose design ideas and adopt them into more than 3 tasks
 
-  - Tech Leader and TiKV Maintainer evaluate team members on a quarterly basis to determine whether members can be promoted to Committer from Reviewer:
+  - Tech Lead and Maintainer evaluate team members on a quarterly basis to determine whether members can be promoted to Committer from Reviewer:
     - Show good technical judgment
-    - Nominated for at least 2 TiKV Maintainers
+    - Nominated for at least 2 Maintainers
```

Comment:
This doesn't make sense. You are confusing nominating with  voting. Also why can't Tech Lead nominate since both Tech Lead and Maintainer evaluate.
In this case, i think one nomination either by Maintainer or Tech Lead is fine.  We still have the PR process for decision making.

Reply:
You are right, I just followed the past writing, I thought the rules were originally like this.

Reply:
See the engine sig constitution: https://github.com/tikv/community/blob/3fad2560b7105637281f6fd0521b9f4d60924693/sig/engine/constitution-zh_CN.md#sig-%E6%B4%BB%E8%B7%83%E8%A7%84%E5%88%99%E6%99%8B%E5%8D%87%E6%9C%BA%E5%88%B6